README.md

```diff
@@ -16,23 +16,21 @@ Currently unsorted, the following is intended to function as a discussion guide
 to developers/contributors.
```

Comment:
If this has to be merged, you'd be better removing these lines, as they are inverse to your pull request.